kivy/_event.pyx

```diff
@@ -1022,6 +1051,31 @@ cdef class EventObservers:
             self.last_callback = new_callback
         return uid
 
+    cdef inline BoundCallback fbind_callback(
+            self, object observer, tuple largs, dict kwargs, int is_ref):
+        '''Similar to bind, except it accepts largs, kwargs that is forwards.
+        is_ref, if true, will mark the observer that it is a ref so that we
+        can unref it before calling.
+        '''
+        cdef BoundCallback new_callback = self.make_callback(
+            observer, largs, kwargs, is_ref)
+
+        if self.first_callback is None:
+            self.last_callback = self.first_callback = new_callback
+        else:
+            self.last_callback.next = new_callback
+            new_callback.prev = self.last_callback
+            self.last_callback = new_callback
```

Comment:
DRY: why not replace these 6 lines with a call of `fbind_existing_callback(new_callback)`

Reply:
thanks a ton, @matham for your great work and for to keep the kivy universe expanding <3

Reply:
Good point, fixed.